docs/community/known-issues.md

```diff
@@ -39,3 +39,26 @@ If you are using [Nginx Proxy Manager](https://nginxproxymanager.com) to open Ho
 
 ### Resolution
 Open setings of Hommar instance on Nginx Proxy Manager and disable 'Block common exploits'.
+
+## 🚨 qBittorrent webUI showing `Unauthorized`
+When creating a qBittorrent tile, The qBittorrent webUI shows an Unauthorized page.
+
+<details>
+  <summary>Screenshot</summary>
+  <img src="./img/qbittorrent_unauthorized.png"/>
```

Comment:
```suggestion
<>

	![](img/qbittorrent_unauthorized.png)

<>
```

This should work. **(I have not tested this)**

Reply:
Hi @walkxcode , thanks for the suggestion. Unfortunately, it does not seem to work.